src/infra/src/table/short_urls.rs

```diff
@@ -162,8 +155,7 @@ pub async fn get(short_id: &str) -> Result<ShortUrlRecord, errors::Error> {
         .filter(Column::ShortId.eq(short_id))
         .into_model::<ShortUrlRecord>()
         .one(client)
-        .await
-        .map_err(|e| Error::DbError(DbError::SeaORMError(e.to_string())))?
+        .await?
```

Comment:
This logic moved to [this trait implementation](https://github.com/openobserve/openobserve/pull/5100/files#r1847373471) so it should now happen automatically when `?` converts the error.